mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -383,10 +413,8 @@ def limits(self):
         :returns: The limits.
         """
         try:
-            energy_limits = self._ho.getLimits()
+            energy_limits = self._ho.getEnergyLimits()
```

Comment:
Ahh, yes for the wavelength object sorry missed that, yes its getWavelengthLimits